pygeos/coordinates.py

```diff
@@ -21,9 +21,9 @@ def apply(geometry, transformation, include_z=False):
     transformation : function
         A function that transforms a (N, 2) or (N, 3) ndarray of float64 to
         another (N, 2) or (N, 3) ndarray of float64.
-    include_z : bool, default False
+    include_z : bool
```

Comment:
Personally, I find this very useful to include the default in the docstring

(yes, that information is also in the signature, but IMO it is useful to have it close to the explanation of the parameter as well)

Reply:
Yes you're right. I changed it back and made it uniform across the rest of the files.